draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -12,17 +12,18 @@ pi: [toc, tocindent, sortrefs, symrefs, strict, compact, comments, inline]
 stand_alone: yes #_
 
 author:
--
-  ins: A. Barth
-  name: Adam Barth
-  organization: Google, Inc
-  uri: https://www.adambarth.com/
 -
```

Comment:
```suggestion
  -
```
identation problem perhaps

Reply:
This is exactly the thing I tried to address. Maybe I failed.